samples/SocialWeather/Startup.cs

```diff
@@ -35,7 +35,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             }
 
             app.UseSockets(o => { o.MapEndpoint<SocialWeatherEndPoint>("/weather"); });
-            app.UseStaticFiles();
```

Comment:
See below comment. It just makes Ctrl-F5 smoother. `UseFileServer` activates default document, so `index.html` works.